src/aspire/source/__init__.py

```diff
@@ -416,10 +416,10 @@ def invert_contrast(self, batch_size=512):
 
         if signal_mean < noise_mean:
             logger.info('Need to invert contrast')
-            scale_factor = -1.0 * np.ones(self.n)
+            scale_factor = np.array(-1.0)
```

Comment:
The caller of the `Multiply` Xform should just be able to pass in a scalar and not have to do this. `__init__` for `Multiply` is the correct place to do any type conversions like this.

Reply:
Done in commit 40b7bfa.